repos/clicommandlineparser.proj

```diff
@@ -23,4 +23,4 @@
   </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))/dir.targets" />
-</Project>
+</Project>
```

Comment:
\<nit> Leave the end-of-file newline--it's expected by various non-Windows tools.

Reply:
Oops my fault, when reverting the changes this slipped in. Will fix.